tests/update_existing_persistent_subscription_with_subscribers.php

```diff
@@ -37,8 +37,8 @@ class update_existing_persistent_subscription_with_subscribers extends TestCase
     private PersistentSubscriptionSettings $settings;
     private Deferred $dropped;
     private ?SubscriptionDropReason $reason = null;
-    private ?Throwable $exception;
-    private Throwable $caught;
+    private ?Throwable $exception = null;
+    private ?Throwable $caught = null;
```

Comment:
`= null` is useless

Reply:
Unfortunately not:
```
Fatal error: Uncaught Error: Typed property Prooph\EventStoreClient\Internal\EventStoreConnectionLogicHandler::$connection must not be accessed before initialization in /home/sasa/code/event-store-client/src/Internal/EventStoreConnectionLogicHandler.php:354
```

So I need to initialize with `null` value, otherwise I got this.

Reply:
Okay, I need this `= null` in some situations and not in others. I check where I can remove it.

Reply:
And that's the reason (from RFC https://wiki.php.net/rfc/typed_properties_v2): `If a typed property does not have a default value, no implicit null default value is implied (even if the property is nullable). Instead, the property is considered to be uninitialized. Reads from uninitialized properties will generate a TypeError (unless __get() is defined, see next section). `

Reply:
I added a few `cleanup` commits to solve this.